src/client/sky.h

```diff
@@ -162,6 +164,7 @@ class Sky : public scene::ISceneNode
 	}
 
 	bool m_visible = true;
+	std::string m_type = "regular";
```

Comment:
Isn't this redundant with `m_sky_params.type`?

Reply:
Thanks for catching that! `m_sky_params.type` wasn't used anywhere before so I thought it's just not passed: I saw the local definitions beginning with "m_" first so I went with those instead. Confirmed it works and will fix it in the patch.